components/layout/flow/inline/mod.rs

```diff
@@ -2079,8 +2079,9 @@ impl IndependentFormattingContext {
         let pbm_physical_offset = pbm_sums
             .start_offset()
             .to_physical_size(container_writing_mode);
-        fragment.content_rect = fragment
-            .content_rect
+        fragment.base.rect = fragment
+            .base
+            .rect
             .translate(pbm_physical_offset.to_vector());
```

Comment:
Unrelated nit, instead of creating a new rect it could be

```suggestion
        fragment.base.rect.origin += pbm_physical_offset.to_vector();
```